specification/migrate/resource-manager/Microsoft.Migrate/stable/2019-05-01/migrate.json

```diff
@@ -0,0 +1,3778 @@
+          "type": "number",
+          "format": "double",
+          "description": "Number of read and write operations per second for the disk.",
+          "readOnly": true
+        },
+        "monthlyStorageCost": {
+          "type": "number",
+          "format": "double",
+          "description": "Estimated aggregate storage cost for a 31-day month for this disk.",
+          "readOnly": true
+        },
+        "recommendedDiskType": {
+          "type": "string",
+          "enum": [
+            "Unknown",
+            "Standard",
+            "Premium",
+            "StandardSSD",
+            "StandardOrPremium"
+          ],
+          "description": "Storage type selected for this disk.",
+          "x-ms-enum": {
+            "name": "AzureDiskType",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "recommendedDiskSize": {
+          "type": "string",
+          "enum": [
+            "Unknown",
+            "Standard_S4",
+            "Standard_S6",
+            "Standard_S10",
+            "Standard_S15",
+            "Standard_S20",
+            "Standard_S30",
+            "Standard_S40",
+            "Standard_S50",
+            "Premium_P4",
+            "Premium_P6",
+            "Premium_P10",
+            "Premium_P15",
+            "Premium_P20",
+            "Premium_P30",
+            "Premium_P40",
+            "Premium_P50",
+            "Standard_S60",
+            "Standard_S70",
+            "Standard_S80",
+            "Premium_P60",
+            "Premium_P70",
+            "Premium_P80",
+            "StandardSSD_E10",
+            "StandardSSD_E15",
+            "StandardSSD_E20",
+            "StandardSSD_E30",
+            "StandardSSD_E40",
+            "StandardSSD_E50",
+            "StandardSSD_E60",
+            "StandardSSD_E70",
+            "StandardSSD_E80",
+            "StandardSSD_E4",
+            "StandardSSD_E6"
+          ],
+          "description": "Recommended Azure size for the disk, given utilization data and preferences set on Assessment.",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "AzureDiskSize",
+            "modelAsString": true
+          }
+        },
+        "gigabytesForRecommendedDiskSize": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gigabytes of storage provided by the recommended Azure disk size.",
+          "readOnly": true
+        },
+        "suitability": {
+          "type": "string",
+          "enum": [
+            "Unknown",
+            "NotSuitable",
+            "Suitable",
+            "ConditionallySuitable",
+            "ReadinessUnknown"
+          ],
+          "description": "Whether this disk is suitable for Azure.",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "CloudSuitability",
+            "modelAsString": true
+          }
+        },
+        "suitabilityExplanation": {
```

Comment:
The difference between suitabilityExplanation and suitabilityDetail is not clear from the property names or descriptions